src/helpers/github.ts

```diff
@@ -19,6 +19,8 @@ export interface Requirements {
   requiredApprovalCount: number;
 }
 
+const allowedRoles = ["COLLABORATOR", "MEMBER", "OWNER"];
```

Comment:
Maybe we could have these being configured through the configuration file itself? Some third parties might want to change the allowed reviewers.